ocis-phoenix/CHANGELOG.md

```diff
@@ -0,0 +1,373 @@
+
+   When a non-existing assets folder is specified, there was only a warning log statement and the
+   service served the builtin assets instead. It is safe to exit the service in such a scenario,
+   instead of serving other assets than specified. We changed the log level to `Fatal` on
+   non-existing assets. Similar for the web config, it was not failing on service level, but only
+   showing an error in the web ui, wenn the specified config file could not be found. We changed the
+   log level to `Fatal` as well.
+
+   https://github.com/owncloud/ocis/ocis-phoenix/pull/76
+   https://github.com/owncloud/ocis/ocis-phoenix/pull/77
+
+* Bugfix - Build docker images with alpine:latest instead of alpine:edge: [#73](https://github.com/owncloud/ocis/ocis-phoenix/pull/73)
+
+   ARM builds were failing when built on alpine:edge, so we switched to alpine:latest instead.
+
+   https://github.com/owncloud/ocis/ocis-phoenix/pull/73
+
+* Change - Hide searchbar by default: [#116](https://github.com/owncloud/product/issues/116)
+
+   Since file search is not working at the moment we decided to hide the search bar by default.
+
+   https://github.com/owncloud/product/issues/116
+   https://github.com/owncloud/ocis/ocis-phoenix/pull/74
+
+* Change - Update Phoenix: [#78](https://github.com/owncloud/ocis/ocis-phoenix/pull/78)
+
+   Updated phoenix from v0.13.0 to v0.14.0
+
+   https://github.com/owncloud/ocis/ocis-phoenix/pull/78
+
+# Changelog for [0.10.0] (2020-07-17)
+
+The following sections list the changes in ocis-phoenix 0.10.0.
+
+[0.10.0]: https://github.com/owncloud/ocis/ocis-phoenix/compare/v0.9.0...v0.10.0
+
+## Summary
+
+* Change - Update Phoenix: [#72](https://github.com/owncloud/ocis/ocis-phoenix/pull/72)
+
+## Details
+
+* Change - Update Phoenix: [#72](https://github.com/owncloud/ocis/ocis-phoenix/pull/72)
+
+   Updated phoenix from v0.12.0 to v0.13.0
+
+   https://github.com/owncloud/ocis/ocis-phoenix/pull/72
+
+# Changelog for [0.9.0] (2020-07-10)
+
+The following sections list the changes in ocis-phoenix 0.9.0.
+
+[0.9.0]: https://github.com/owncloud/ocis/ocis-phoenix/compare/v0.8.1...v0.9.0
+
+## Summary
+
+* Bugfix - Allow silent refresh of access token: [#69](https://github.com/owncloud/ocis-konnectd/issues/69)
+* Change - Update Phoenix: [#70](https://github.com/owncloud/ocis/ocis-phoenix/pull/70)
+
+## Details
+
+* Bugfix - Allow silent refresh of access token: [#69](https://github.com/owncloud/ocis-konnectd/issues/69)
+
+   Sets the `X-Frame-Options` header to `SAMEORIGIN` so the oidc client can refresh the token in
+   an iframe.
+
+   https://github.com/owncloud/ocis-konnectd/issues/69
+   https://github.com/owncloud/ocis/ocis-phoenix/pull/69
+
+* Change - Update Phoenix: [#70](https://github.com/owncloud/ocis/ocis-phoenix/pull/70)
+
+   Updated phoenix from v0.11.1 to v0.12.0
+
+   https://github.com/owncloud/ocis/ocis-phoenix/pull/70
+
+# Changelog for [0.8.1] (2020-06-29)
```

Comment:
Codacy found an issue: [[no-shortcut-reference-link] Use the trailing `[]` on reference links](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6202733)